src/components/HierarchicalMenu.vue

```diff
@@ -1,55 +1,85 @@
 <template>
-  <div>
-    <slot v-bind="state">
-      <json-tree :level="2" :data="state"></json-tree>
+  <div
+    v-if="state"
+    :class="[suit(''), !canRefine && suit('', 'noRefinement')]"
+  >
+    <slot
+      :items="items"
+      :can-refine="canRefine"
+      :can-toggle-show-more="canToggleShowMore"
+      :is-showing-more="isShowingMore"
+      :refine="state.refine"
+      :create-URL="state.createURL"
+      :toggle-show-more="toggleShowMore"
+    >
+      <ais-hierarchical-menu-list
+        :items="items"
+        :level="0"
+        :suit="suit"
+        :refine="state.refine"
+        :createURL="state.createURL"
+      />
+
+      <button
+        v-if="showMore"
+        :class="[suit('showMore'), !canToggleShowMore && suit('showMore', 'disabled')]"
+        :disabled="!canToggleShowMore"
+        @click.prevent="toggleShowMore"
+      >
+        <slot name="showMoreLabel" :is-showing-more="isShowingMore">
+          {{ isShowingMore ? 'Show less' : 'Show more' }}
+        </slot>
+      </button>
     </slot>
   </div>
 </template>
 
 <script>
-import JsonTree from 'vue-json-tree'; // todo: remove
-import algoliaComponent from '../component';
 import { connectHierarchicalMenu } from 'instantsearch.js/es/connectors';
+import algoliaComponent from '../component';
+import HierarchicalMenuList from './HierarchicalMenuList.vue';
 
 export default {
-  components: { 'json-tree': JsonTree },
   mixins: [algoliaComponent],
+  components: {
+    'ais-hierarchical-menu-list': HierarchicalMenuList,
```

Comment:
if you imported it as `AisHierarchicalMenuList`, the mapping would be unnecessary and you could have written `components: { AisHierarchicalMenuList }`

Reply:
Yep indeed but I prefer to keep the same name for the import. Since this widget is a private API we can remove the prefix and use the component as it is `components: { HierarchicalMenuList }`. WDYT?